pio-dist/pio-eventserver.conf

```diff
@@ -0,0 +1,18 @@
+# Upstart service configuration
+env USER=ubuntu
+env USER_HOME=/home/ubuntu
+
+env LOGFILE=/opt/PredictionIO-0.9.5/stdout.log
+
+start on runlevel [2345]
+stop on runlevel [016]
+
+setuid ubuntu
+chdir /opt/PredictionIO-0.9.5/engines/MyGuardianTestEngine-1
+
+script
+
+COMMAND="/opt/PredictionIO-0.9.5/bin/pio eventserver"
+echo "$COMMAND" >/opt/PredictionIO-0.9.5/cmd.txt
+$COMMAND >$LOGFILE 2>&1
```

Comment:
This is fixing an error that I introduced, but to avoid restarting the service destroying the existing log file:
$COMMAND >>$LOGFILE 2>&1